server/pulp/server/controllers/repository.py

```diff
@@ -202,7 +203,7 @@ def create_repo(repo_id, display_name=None, description=None, notes=None, import
     """
 
     # Prevalidation.
-    if not isinstance(distributor_list, (list, tuple)):
+    if distributor_list is not None and not isinstance(distributor_list, (list, tuple)):
```

Comment:
just as another option that's a bit more readable, this could be
```
 if not isinstance(distributor_list, (list, tuple, type(None))):
```

or

```
 import types
 if not isinstance(distributor_list, (list, tuple, types.NoneType)):
```